### ISSUE TITLE
Il progetto nasce durante la pandemia Covid-19 e attraverso l’uso di strumenti telematici, con l’att

### ISSUE THREAD

**Issue by ehack-italy** [emergenzeHack/covid19italia_segnalazioni#2842]
<pre><yamldata>
Da_chi_offerta: Fondazione Arché
Descrizione: Il progetto nasce durante la pandemia Covid-19 e attraverso l’uso di
  strumenti telematici, con l’attivazione di un’équipe di professionisti e il supporto
  di volontari, mira a prevenire la nascita di problemi nel percorso scolastico (o
  il suo abbandono), garantendo un supporto educativo adeguato, favorendo le competenze
  genitoriali e monitorando il benessere psico-fisico di minori e genitori attraverso
  azioni di ascolto e sostegno psicologico.
Destinatari: Famiglie messe in difficoltà nel seguire il percorso educativo dei figli
Link: https://arche.it/cosa-facciamo/sostegno-alle-famiglie/smart-together/
Natura: solidale
Tipo_di_soggetto: pubblico
Titolo: Smart Together
</yamldata></pre>

**Comment by marticorti** [comment]
@cristigalas Questo servizio è attivo in tre regioni differenti, perciò ho inserito la posizione "regione_manuale: tutte". Pensi che possa andar bene come soluzione?

**Comment by cristigalas** [comment]
@marticorti no, meglio toglierla e aggiungere la Posizione con la sede della fondazione, cioè dell’ente promotore.

**Comment by marticorti** [comment]
@cristigalas ok, fatto!
